### PR TITLE
Browserview settings window

### PR DESCRIPTION
**Pull request by Willyfrog**
Please provide the following information:

**Summary**
This PR is mainly for ensuring the settings page works as expected, as a result some other parts have improved. But some effects on the main page doesn't still happen (like changing the tabs in the main window when adding/removing a server, although server is there and attached)

for now use the settings page to move through tabs

**Additional Notes**
It is based on [previous PR work](https://github.com/mattermost/desktop/pull/1358), please review that other before this one. once that one is merged, changes will go down by a lot :)

### CODE REVIEW

**Review by nevyangelova — APPROVED** [review]

**Comment by Willyfrog** [comment]
PR updated (don't really get how it ended with merge conflicts if it was based from the previous branch)

**Review by devinbinnie — APPROVED** [review]
Looks good, just a couple comments

File: src/main/menus/app.js [hunk 1]
```diff
@@ -286,8 +293,8 @@ function createTemplate(mainWindow, config, isDev) {
   return template;
 }
 
-function createMenu(mainWindow, config, isDev) {
-  return Menu.buildFromTemplate(createTemplate(mainWindow, config, isDev));
+function createMenu(mainWindow, config) {
```
[the remark this answers is not in the document]
Comment (Willyfrog) [reply src/main/menus/app.js:296]: i already did that in the next PR: https://github.com/mattermost/desktop/pull/1378/files#diff-85b74de09048cfb3bdb71b9ed2f60e3c2e6e421f6b23d3338d45c187b5b1ea71R296

File: src/main/menus/tray.js [hunk 2]
```diff
@@ -46,16 +36,8 @@ function createTemplate(mainWindow, config, isDev) {
...
-  return Menu.buildFromTemplate(createTemplate(mainWindow, config, isDev));
-}
-
-function showOrRestore(window) {
-  if (window.isMinimized()) {
-    window.restore();
-  } else {
-    window.show();
-  }
+function createMenu(mainWindow, config) {
```
[the remark this answers is not in the document]
Comment (Willyfrog) [reply src/main/menus/tray.js:39]: same as above

**Pull request merged**